src/common/selftest/selftest_heater.h

```diff
@@ -0,0 +1,42 @@
+// selftest_heater.h
+#pragma once
+
+#include <inttypes.h>
+#include "selftest_MINI.h"
+
+typedef struct _selftest_heater_config_t {
```

Comment:
why C struct?

Reply:
why not? i need simple define entire structure as constant (seperate the members from the class). the structure is used in constructor and in static method estimate (to calculate total time of the test).